### PR TITLE
Add title page

### PR DESCRIPTION
**Pull request by Navneet-cmd**
Add a title page to the all-about-git about us page.

### CODE REVIEW

**Comment by Navneet-cmd** [comment]
Thank you for your review and comments, Will work on the changes as requested. :)

File: about.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>'About Us Page'</title>
+    <link rel="stylesheet" href="aboutpage.css"
```
Comment (rmklaus12) [inline about.html:7]: There is a missing closing bracket here

**Comment by rmklaus12** [comment]
There is one small change to be made in the about.html file. Also, the aboutpage.md file needs to be removed from the PR. You can do this using the `git commit rebase` command - with additional attributes that you will need to research :-)

**Comment by Navneet-cmd** [comment]
Thank you for your review and comment, will work on the issues and fix as recommended :)

**Pull request merged**